demo/send/mailingLists/27cda576-4a60-4621-b515-8489b396a300.json

```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":2,"MailingList":{"Id":"27cda576-4a60-4621-b515-8489b396a300","MailingListCode":"2947879","Name":"Subscribers from https://adotest-cm.sitecoredemo.com/","MailingListName":"Subscribers from https://adotest-cm.sitecoredemo.com/","ActiveMemberCount":2,"BouncedMemberCount":0,"RemovedMemberCount":0,"UnsubscribedMemberCount":0,"Status":0,"CreatedBy":"7dc7a9bd0a14427f894a66d8f7a938db","CreatedOn":"/Date(1643747091280-0000)/","UpdatedOn":"/Date(-62135596800000-0000)/","DeletedMemberCount":1,"IsSupressionList":false},"Entity":null}
```

Comment:
is this "Subscribers from https://adotest-cm.sitecoredemo.com/" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?